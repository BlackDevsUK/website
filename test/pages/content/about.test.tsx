import { render } from "@testing-library/react";
import About from "../../../pages/content/about";

describe("About Page", () => {
  it("should match snapshot", () => {
    const { asFragment } = render(<About />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
