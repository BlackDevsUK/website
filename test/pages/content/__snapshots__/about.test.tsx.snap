// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page should match snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="relative pt-6 px-4 sm:px-6 lg:px-8"
    >
      <nav
        aria-label="Global"
        class="relative flex items-center justify-between sm:h-10 lg:justify-start"
      >
        <div
          class="flex items-center flex-grow flex-shrink-0 lg:flex-grow-0"
        >
          <div
            class="flex items-center justify-between w-full md:w-auto"
          >
            <a
              href="#"
            >
              <span
                class="sr-only"
              >
                BDUK
              </span>
              <div
                style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
              >
                <div
                  style="box-sizing: border-box; display: block; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    role="presentation"
                    src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNjIiIGhlaWdodD0iNDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                    style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
                  />
                </div>
                <img
                  alt="Black Devs UK log"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="visibility: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
              </div>
            </a>
            <div
              class="-mr-2 flex items-center md:hidden"
            >
              <button
                aria-expanded="false"
                class="bg-white rounded-md p-2 inline-flex items-center justify-center text-gray-400 hover:text-gray-500 hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-indigo-500"
                type="button"
              >
                <span
                  class="sr-only"
                >
                  Open main menu
                </span>
                <svg
                  aria-hidden="true"
                  class="h-6 w-6"
                  fill="none"
                  stroke="currentColor"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M4 6h16M4 12h16M4 18h16"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                    stroke-width="2"
                  />
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="hidden md:block md:ml-10 md:pr-4 md:space-x-8"
        >
          <a
            class="font-medium text-gray-500 hover:text-gray-900"
            href="/content/home"
          >
            Home
          </a>
          <a
            class="font-medium text-gray-500 hover:text-gray-900"
            href="/content/about"
          >
            About Us
          </a>
          <a
            class="font-medium text-gray-500 hover:text-gray-900"
            href="/content/contact"
          >
            Contact us
          </a>
        </div>
      </nav>
    </div>
    <div
      class="md:container md:mx-auto"
    >
      <h1
        class="font-bold text-5xl text-center mt-9"
      >
        About Black Devs UK
      </h1>
      <article>
        <p
          class="home-text"
        >
          I'm baby chambray four dollar toast man bun blog roof party gastropub actually austin tumblr. Venmo brunch migas, try-hard thundercats food truck viral blue bottle locavore pork belly forage kickstarter humblebrag. Shaman post-ironic ennui freegan yr jean shorts iPhone sartorial marfa narwhal stumptown fashion axe. Dreamcatcher bespoke post-ironic vice pop-up celiac umami, vaporware jianbing. Enamel pin ennui banjo tumeric edison bulb taxidermy, iPhone brooklyn prism mustache ramps narwhal pour-over air plant. Small batch sriracha fixie pug, subway tile asymmetrical crucifix tousled meggings synth brooklyn. Try-hard lumbersexual meditation hammock polaroid tbh.
        </p>
        <p
          class="home-text"
        >
          I'm baby chambray four dollar toast man bun blog roof party gastropub actually austin tumblr. Venmo brunch migas, try-hard thundercats food truck viral blue bottle locavore pork belly forage kickstarter humblebrag. Shaman post-ironic ennui freegan yr jean shorts iPhone sartorial marfa narwhal stumptown fashion axe. Dreamcatcher bespoke post-ironic vice pop-up celiac umami, vaporware jianbing. Enamel pin ennui banjo tumeric edison bulb taxidermy, iPhone brooklyn prism mustache ramps narwhal pour-over air plant. Small batch sriracha fixie pug, subway tile asymmetrical crucifix tousled meggings synth brooklyn. Try-hard lumbersexual meditation hammock polaroid tbh.
        </p>
        <p
          class="home-text"
        >
          I'm baby chambray four dollar toast man bun blog roof party gastropub actually austin tumblr. Venmo brunch migas, try-hard thundercats food truck viral blue bottle locavore pork belly forage kickstarter humblebrag. Shaman post-ironic ennui freegan yr jean shorts iPhone sartorial marfa narwhal stumptown fashion axe. Dreamcatcher bespoke post-ironic vice pop-up celiac umami, vaporware jianbing. Enamel pin ennui banjo tumeric edison bulb taxidermy, iPhone brooklyn prism mustache ramps narwhal pour-over air plant. Small batch sriracha fixie pug, subway tile asymmetrical crucifix tousled meggings synth brooklyn. Try-hard lumbersexual meditation hammock polaroid tbh.
        </p>
      </article>
    </div>
  </div>
</DocumentFragment>
`;
